hydromt_fiat/api/hydromt_fiat_vm.py

```diff
@@ -140,6 +140,45 @@ def run_hydromt_fiat(self):
         region = self.data_catalog.get_geodataframe("area_of_interest")
         self.fiat_model.build(region={"geom": region}, opt=config_yaml.dict())
 
+        # Update exposure dataframe
+        buildings_gdf, roads_gdf = self.update_exposure_db(config_yaml)
+        return buildings_gdf, roads_gdf
+
+    def update_model(self, parameter):
+
+        #Update config yaml
+        self.save_data_catalog()
+        config_yaml = self.build_config_yaml()
+
+        # Update parameter with user-input
+        if isinstance(parameter,list):
+            for item in parameter:
+                if "Finished Floor Height" in item:
+                    self.new_ground_floor_height(config_yaml)
+                elif "Additional Attributes" in item :
+                    self.new_additional_attributes(config_yaml)
+                elif"Ground Elevation" in item:
+                    self.new_ground_elevation(config_yaml)
+                elif "Max Potential Damages" in item :
+                    self.new_max_potential_damages(config_yaml)
+        else:
+            if parameter == "Finished Floor Height" :
+                self.new_ground_floor_height(config_yaml)
+            elif parameter == "Additional Attributes" :
+                self.new_additional_attributes(config_yaml)
+            elif parameter == "Ground Elevation" :
+                self.new_ground_elevation(config_yaml)
+            elif parameter == "Max Potential Damages" :
+                self.new_max_potential_damages(config_yaml)
```

Comment:
Small suggestion to reduce code
```suggestion
        if isinstance(parameter, str):
            parameter = [parameter]
        for item in parameter:
            if "Finished Floor Height" in item:
                self.new_ground_floor_height(config_yaml)
            elif "Additional Attributes" in item :
                self.new_additional_attributes(config_yaml)
            elif"Ground Elevation" in item:
                self.new_ground_elevation(config_yaml)
            elif "Max Potential Damages" in item :
                self.new_max_potential_damages(config_yaml)
```